Samples/src/OpenQasmReader/Parser.cs

```diff
@@ -305,7 +305,7 @@ private static void ParseGateCall(IEnumerator<string> token, StringBuilder build
                 }
                 else if (withinParentheses)
                 {
-                    doubles.Add(ParseCalculation(token, COMMA, CLOSE_PARANTHESES));
+                    doubles.Add(ParseCalculation(token, $"{COMMA} ", CLOSE_PARANTHESES));
```

Comment:
Would it make sense to make this a constant of its own?

Reply:
I'll let you do that later ;p